projects/cashmere/src/lib/form-field/hc-form-field.component.spec.ts

```diff
@@ -74,12 +74,12 @@ describe('HcFormFieldComponent', () => {
             expect(labelElement.textContent).toMatch(/^hola$/);
         });
 
-        it('support type attribute', () => {});
+        // it('support type attribute', () => {});
```

Comment:
We should remove empty commented out code here

Reply:
I wasn't sure if some of the empty or commented unit tests were stubs that people thought should be in but never got to.  Happy to remove the commented or `xit` items if you think that's best.

Reply:
I'll go ahead and remove all the commented tests, but I'll leave the `xit` ones assuming they are stubs.